application/src/main/webapp/vue-app/components/ExoChatContact.vue

```diff
@@ -183,10 +193,10 @@ export default {
       this.isOnline = false;
     },
     getProfileLink() {
-      if (this.app && this.type === 'u') {
```

Comment:
why did you remove the check on this.app ?

Reply:
app is undefined